starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+body{
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+	margin: 0;
+	line-height: 1.5;
+}
+
+header{
+	background-image: url(../images/apple-pie.jpg);
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: 0 0;
```

Comment:
Your background positioning works well :)
You might also want to take a look at the ``center`` value for ``background-position``, in most of the cases it's more "adaptive".